### PR TITLE
Added navbar

### PR DESCRIPTION
**Pull request by Pedro-Pan**
image

Hey does this look good, anything to change ?

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
+  <div class="container-fluid">
+    <%= link_to "#", class: "navbar-brand" do %>
+      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
+    <% end %>
+
```
Comment (1zach) [inline app/views/shared/_navbar.html.erb:6]: Looks great

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
+  <div class="container-fluid">
+    <%= link_to "#", class: "navbar-brand" do %>
+      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
```
Comment (Pedro-Pan) [inline app/views/shared/_navbar.html.erb:4]: Love the usage of image_tag instead of a img

Comment (1zach) [inline app/views/shared/_navbar.html.erb:4]: Can we change this photo?

**Review by 1zach — CHANGES_REQUESTED** [review]

**Review by 1zach — APPROVED** [review]
Perfect!

**Pull request merged**